Scripts/RemediationScripts/Remediate-ConfigureWAFOnAzureFrontDoorCDN.ps1

```diff
@@ -1,810 +0,0 @@
-﻿<###
```

Comment:
I see just removal of previous code but no addition. Is this expected?

Reply:
[Remediate-ConfigureWAFPolicyForFrontDoorCDN] is already merged by Richa